getzillowhtml.py

```diff
@@ -1,14 +1,11 @@
-
```

Comment:
i put all these blank spaces in at the top because otherwise PyCharm gives me "warnings" that it "expected 2 blank lines at start of file" kind of shit and i hate seeing the warning icons appear when i have the code open, so that's why i comply and it makes it look nicer.

Reply:
its actually PEP 8 E302 warnings it gives. so i guess pycharm enforces a specific style guide, which requires two spaces. something you wouldn't really know or care about mr. sloppy Joe

Reply:
@AmritPatel

Reply:
@AmritPatel see [https://www.python.org/dev/peps/pep-0008/](PEP 8 -- Style Guide for Python Code)

Reply:
@AmritPatel other warnings I'm getting all over the place:
1. Function name should be lowercase
2. Argument name should be lowercase
3. Variable in function should be lowercase

I like how we've named the functions, variables within functions, and the function arguments with lower/uppercase, but PEP 8 has specific naming conventions... how to treat? ignore, or follow PEP8?